server/src/frontend/components/duck/operations.js

```diff
@@ -70,7 +70,10 @@ const getClientComponents = (onSuccess) => (dispatch) => {
                         acc[clientCode].subComponents[component.id] = component;
                         return acc;
                     }, {})
-                dispatch(actions.receiveComponents(parents))
+                const sortedParents = Object.fromEntries(
+                    Object.entries(parents).sort(([keyA], [keyB]) => keyA.localeCompare(keyB))
+                )
+                dispatch(actions.receiveComponents(sortedParents))
```

Comment:
suggest to make as there: https://github.com/octopusden/octopus-dms-service/pull/43/files#diff-8253aae371b91defaa846330317519c0b1f48460b48dda60e0f2c41040b64519L105

1. Same approach
2. sort ignoring case

Reply:
Sorry, missed it, I'll update it